frontend/vite.config.ts

```diff
@@ -61,6 +105,13 @@ export default defineConfig(({ mode }) => {
                 // Just for Vite: we copy public assets to src/assets, we need to alias it to the correct path
                 public: resolve(__dirname, 'src/assets'),
                 products: resolve(__dirname, '../products'),
+                // Alias for docs/onboarding directory
+                onboarding: resolve(__dirname, '../docs/onboarding'),
+                // Alias for components/Docs/OnboardingContentWrapper (used in docs)
+                'components/Docs/OnboardingContentWrapper': resolve(
```

Comment:
This one is a bit insane. Do we really need it to be `components/Docs`? It feels so wrong 🤔

Reply:
Let me patch up the website PR first for more context here :) you can tell me how crazy these changes are, I tend to be a little crazy sometimes

Reply:
Okay, have a look at [this PR](https://github.com/PostHog/posthog.com/pull/13856) as well.

If you look at [this line](https://github.com/PostHog/posthog.com/pull/13856/files#diff-5c260fa67a6fec6f6a145897f196aa2a4165c6b4f7b4be909b638c489dd0d2c4R34), we can add the alias here or in the website repo.

I can probably work to not add this here and pile some more onto good ol' gatsby haha